Ported/combat-bees/Assets/Scripts/Systems/FoodSpawnerSystem.cs

```diff
@@ -8,11 +8,16 @@
 [BurstCompile]
 partial struct FoodSpawnerSystem : ISystem
 {
+    private EntityQuery NestQuery;
+    
     [BurstCompile]
     public void OnCreate(ref SystemState state)
     {
+        NestQuery = state.GetEntityQuery(/*typeof(Faction), */typeof(Area));
```

Comment:
We should be using SystemAPI.QueryBuilder it seems.  The DOTS tutorial isn't super helpful in that it only uses GetEntityQuery in SystemBase, which is ok because SystemBase is managed code.  It seems to work but I'm not sure what the side effects are (probably that this method doesn't actually compile with Burst).  This already creates an error in MovementSystem.
```suggestion
        NestQuery = SystemAPI.QueryBuilder().WithAll<Area>().Build();
```